Tella/Scenes/Uwazi/Utils/UwaziEntityParser.swift

```diff
@@ -90,7 +90,7 @@ class UwaziEntityParser: UwaziEntityParserProtocol {
                                               selectValues:[] ,
                                               name: $0.name)
                 
-            case .dataTypeSelect, .dataTypeMultiSelect:
+            case .dataTypeSelect:
```

Comment:
What do you think if we remove `dataTypeMultiSelect` only here when adding the prompt and keep it when handling ?